### PR TITLE
343 Added dropshadow to the bookmark icon of a post

### LINKED ISSUE #343: [343] Bookmark Invisible
**Bug display and/or description**
image

White background makes bookmark icon invisible

image

**Reproduce**
- [ ] Create post with white banner
- [ ] Go to homepage
- [ ] Bookmark invisible

**Branch**
343-hotfix-bookmark-invisible

### PR DESCRIPTION
**Pull request by sribadal**
closes #343 

<br />

 **What is it supposed to do**  
I added drop shadow to the bookmark icon of a post/thumbnail. If the background of a post/thumbnail is white, the bookmark icon will still be seen.
image

<br />

 **How can we test your branch**  
1. Create a post with a white background.
2. Make sure your post will be shown as a large thumbnail.
image

<br />

 **Additional notes**  
The small thumbnail uses a different design, so the bookmark icon is always seen regardless of the background color.

<br />

**1. General checklist**
- [x] a. Tested happy flow.
- [x] b. Tested unhappy flow.
- [x] c. No unexpected exceptions.
- [x] d. No code with high complexity. (Big O)
- [x] e. EOF newline.
- [x] f. No linter warnings or errors.

<br />

**2. If relevant, front-end checklist**
- [x] a. Responsive. (Desktop all the way to mobile 320px)
- [x] b. Using SCSS variables where relevant.
- [x] c. Relevant class names
- [x] d. No modified global SCSS properties, except if necessary and fully tested the impact.
- [x] e. Checked [caniuse](https://caniuse.com) for more modern CSS properties.

<br />

**3. If relevant, back-end checklist**
- [ ] a. No unhandled promise rejections.
- [ ] b. Seeders still functional.
- [ ] c. No primary-key violations.
- [ ] d. Normalization, at least [3NF](https://en.wikipedia.org/wiki/Database_normalization).
- [ ] e. Used guards (middleware) where necessary and tested if route is guarded.

<br />

**4. If relevant, test these browsers**
- [x] a. Chrome
- [x] b. Firefox
- [x] c. Opera
- [x] d. Edge
- [ ] e. Safari

---

**PR Rules**  
*1. Add relevant labels.*  
*2. Select at least two reviewers.*  
*3. Assign all reviewers allowed to merge the branch, including yourself.*  
*4. If relevant to the sprint, put related issue in 'Awaiting Approval' column on project board.*  
*5. Add to the corresponding [milestone](https://github.com/AjobK/Seaqull/milestones).*

<br />

**Reviewer info**  
*1. You can use aliases to refer to specific PR checklist items. For example 'Hey @JohnDoe, for this line please check 3.a and 1.d.'*  
*2. Where possible, please use the suggestions feature in github so the author has a better idea of the solution you had in mind.*   

<br />

> This form should be filled in by the author of this PR.

### CODE REVIEW

File: frontend/src/components/postsBlockLarge/postsBlockLarge.scss [hunk 1]
```diff
@@ -43,6 +43,11 @@ $post-padding: 20px;
 			padding: 4px;
 			margin: $post-padding - 4px;
 			@include bookmark();
+
+			&Icon {
+				filter: drop-shadow(0 0 2px rgba(0, 0, 0, 0.45));
+				//text-shadow: 0 0 3px rgba(0, 0, 0, 0.35);
```
Comment (AjobK) [inline frontend/src/components/postsBlockLarge/postsBlockLarge.scss]: Please remove this comment if it is not necessary 😄 

Otherwise looks good!

**Review by AjobK — CHANGES_REQUESTED** [review]

File: frontend/src/components/postsBlockLarge/postsBlockLarge.scss (hunk 1 above)
Comment (sribadal) [reply frontend/src/components/postsBlockLarge/postsBlockLarge.scss]: Yes you are right man

Comment (sribadal) [reply frontend/src/components/postsBlockLarge/postsBlockLarge.scss]: It's gone now!

**Review by AjobK — APPROVED** [review]
Well done! 😄

**Pull request merged**